.changeset/three-nights-decide.md

```diff
@@ -0,0 +1,9 @@
+---
+'@solana/transaction-messages': minor
+'@solana/instruction-plans': minor
+'@solana/transactions': minor
+'@solana/signers': minor
+'@solana/kit': minor
```

Comment:
Flagging this - I think this is debatable on minor or major. `TransactionMessage` is compatible with `BaseTransactionMessage`, so app code isn't going to break. But if you have functions/types that use `BaseTransactionMessage` you may need to change them to use `TransactionMessage` in order to be compatible with our helper functions.